addons/website/static/src/scss/bootstrap_overridden.scss

```diff
@@ -134,7 +139,17 @@ $h6-margin-bottom: o-website-value('h6-margin-bottom') or $headings-margin-botto
 
 $headings-color: o-color('o-cc1-headings') !default;
 
-$lead-font-size: 1.125rem !default;
+$font-weight-bold: 600 !default;
+
+$display-font-sizes: (
+  1: 6.125rem,
+  2: 5.375rem,
+  3: 4.875rem,
+  4: 4.25rem,
+  5: 3.75rem,
+  6: 3.5rem
+) !default;
```

Comment:
I just had a look here and I wonder why you need to increase the values of display-X classes ? We have [another task](https://github.com/odoo/odoo/pull/129791/files#diff-73090a6cb3708fdfbcfda0141b65c0ad7017af7ba94883e8c3f1d750677df5d7R1964-R1971) where we will use those classes and IMO those class are already big enough.. Do you need text bigger than 5rem ?

Reply:
Thanks for the check and the fast answer :)